docs/developer/releasing.md

```diff
@@ -62,30 +62,39 @@ export GPG_TTY=$(tty)
 
 ## Performing Releases
 
-The lifetime of a release branch is sheparded by a single person, picked from a
-rota of project maintainers and contributors. The release shepard is
+The lifetime of a release branch is shepherded by a single person, picked from a
+rota of project maintainers and contributors. The release shepherd is
 responsible for managing all releases that compose the release branch.
 
-For a new release branch, the release shepard will:
+For a **new release candidate**, the release shepherd will:
```

Comment:
Personally, I preferred the older instructions, because it talked about a release branch as a unit: a stream of releases for the same major/minor version tuple. 

I'm concerned saying "for a new release candidate" may be confusing, because the same minor release branch may contain any number of release candidates, depending on how many bugs are discovered. 

Maybe we need a new term here to refer to these branches? "Release streams?"

Reply:
My reasoning for splitting this into sections of "new release candidate", "stable release" and "patch releases" is that people will likely do them at three separate times. When someone finishes with the "new release candidate" section they are supposed to pause for a few days and then resume from the next section on "stable release". It makes it a little easier to find where to resume, instead of looking for the correct point in a list.

If we could make the headings something like "Step 1: New release candidate", "Step 2: Stable release" and "Step 3: Patch releases", maybe that'd make it more clear that those are different steps of the same continuous process?

In my case, I suppose the "unit" is the Github release and the outcome of each of the three sections is a "release" as in a "Github release". I agree there is value in thinking of the release branch itself as a unit though. Maybe instead of "Performing Releases", this whole section should be called "Release branch lifecycle"?